Modules/hpe3par_snapshot.py

```diff
@@ -117,11 +117,21 @@
     description:
       - "Specifies a snapshot volume name."
     required: true
+  schedule_name:
+    description:
+      - "Name of the schedule."
+    required: true
+  task_freq:
+    description:
+      - "Frequency of the schedule to be created."
+    required: false
```

Comment:
Please make sure all the parameters are listed in the documentation

Reply:
okay